Changes to Homepage

js-ts opened the pull request:
Site can be viewed at [site](https://cranky-lumiere-5df5c6.netlify.app/) 
Changes Made :
+ Added Favicon
+ Changed logo size
+ Added Top Banner
+ Added Banner
+ Added Points Section
+ Added Component Cards
+ Added Adopters Section
+ Changed controlplane.png
Will Add :
+ Blog
In the Next PR

theofpa reviewed (changes requested):
Can we skip the files about building a docker image for mkdocs? It's better to keep the repo clean from the way of publishing it and focus on the content.

yuzisun commented on overrides/home.html:
```diff
@@ -0,0 +1,263 @@
+            for Trusted AI
+          </h2>
+          <p style="display: block">
+            <a href="{{ 'getting_started/' | url }}" class="md-button md-button--primary">
```
/getting_started/get_started/gc

yuzisun commented on overrides/home.html:
```diff
@@ -0,0 +1,263 @@
+          <b>serverless inference workload</b> with <b>Autoscaling</b> including <b>Scale to Zero on GPU</b>.
+        </li>
+        <li class="task-list-item">
+          <input type="checkbox" checked disabled /><span class="task-list-indicator"></span>Provides <b>high scalability</b>, <b>density packing</b> and <b>intelligent routing</b> using <b>ModelMesh</b>
```
nit: add period at the end.

yuzisun commented:
/lgtm
/approve

yuzisun commented:
Really awesome work! Thanks @js-ts !!

js-ts commented:
@theofpa Deleted All the docker and deployment files

Deleted .dockerignore
Deleted .gitlab-ci.yml
Deleted Dockerfile
Deleted amplify.yml
Deleted docker-compose.yml

theofpa commented:
Thank you @js-ts great stuff!

theofpa commented:
/lgtm
/appprove
